tracks/csharp/exercises/raindrops/README.md

```diff
@@ -0,0 +1,78 @@
+### Reasonable solutions
+#### LINQ
+
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class Raindrops
+{
+    public static string Convert(int number)
+    {
+        Dictionary<int, string> dict = new Dictionary<int, string>()
+        {
+            [3] = "Pling",
+            [5] = "Plang",
+            [7] = "Plong",
+        };
+
+        var factors = dict
+            .Where(kv => number % kv.Key == 0)
+            .Select(kv => kv.Value)
+            .DefaultIfEmpty(number.ToString());
+
+        return String.Join(string.Empty, factors);
+    }
+}
+```
+
+#### Extreme LINQ
+```csharp
+using System.Linq;
+
+public static class Raindrops
+{
+    public static string Convert(int number)
+        => new (string sound, int value)[] {("Pling", 3), ("Plang", 5), ("Plong", 7)}
+            .Where(p => number % p.value == 0).Select(p => p.sound)
+            .DefaultIfEmpty(number.ToString()).Aggregate((sounds, sound) => sounds + sound);
+}
+```
+#### Non-LINQ
+```csharp
+using System;
+
+public static class Raindrops
+{
+    public static string Convert(int number)
+    {
+        string sound = string.Empty;
+
+        if (number % 3 == 0) sound += "Pling";
+        if (number % 5 == 0) sound += "Plang";
+        if (number % 7 == 0) sound += "Plong";
+
+        return String.IsNullOrEmpty(sound) ? number.ToString() : sound;
```

Comment:
I believe the C# coding conventions suggest to use `string.IsNullOrEmpty` (lowercase string).